frontend/package.json

```
@@ -6,10 +6,12 @@
     "start": "webpack-dev-server --mode development",
     "check": "tsc -p tsconfig.json --noEmit --skipLibCheck",
     "lint": "eslint --max-warnings=0 --ext=.ts,.tsx,.js,.jsx .",
-    "lint:style": "stylelint \"**/*.scss\" \"**/*.pcss\" \"**/*.css\" \"iframe.html\"",
+    "lint:style": "stylelint '**/*.scss' '**/*.pcss' '**/*.css' 'iframe.html'",
     "test": "jest",
     "test:coverage": "jest --coverage",
-    "prettier": "prettier --write \"./**/*.{js,jsx,ts,tsx,scss}\"",
+    "prettier": "prettier --write './**/*.{js,jsx,ts,tsx,scss}'",
+    "extract-messages": "formatjs extract --out-file=./extracted-messages/messages.json './app/**/*.js' './app/**/*.jsx' './app/**/*.tsx' './app/**/*.ts'",
```

formatjs cli using "glob" and we can write it simpler.
```suggestion
    "extract-messages": "formatjs extract --out-file=./extracted-messages/messages.json './app/**/*.{js,jsx,ts,tsx}'",
```